### PR TITLE
Magento

### PR DESCRIPTION
**Pull request by designbykhalid**
Remove skeletor.install.js including references to any JS and patterns.

### CODE REVIEW

File: orko/package.json [hunk 1]
```diff
@@ -6,18 +6,12 @@
...
-        "@babel/core": "^7.1.6",
-        "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-        "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-        "@babel/preset-env": "^7.1.6",
         "@deg-skeletor/core": "^1.0.0",
         "@deg-skeletor/plugin-copy": "^1.0.2",
         "@deg-skeletor/plugin-express": "^1.0.0",
-        "@deg-skeletor/plugin-patternlab": "^3.1.0",
         "@deg-skeletor/plugin-postcss": "^1.0.0",
         "@deg-skeletor/plugin-rollup": "^2.0.2",
```
Comment (aladage) [inline orko/package.json:12]: This can be removed

File: orko/package.json [hunk 2]
```diff
@@ -6,18 +6,12 @@
     "repository": "{{repo}}",
     "license": "{{license}}",
     "devDependencies": {
-        "@babel/core": "^7.1.6",
-        "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-        "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-        "@babel/preset-env": "^7.1.6",
         "@deg-skeletor/core": "^1.0.0",
         "@deg-skeletor/plugin-copy": "^1.0.2",
         "@deg-skeletor/plugin-express": "^1.0.0",
```
Comment (aladage) [inline orko/package.json]: If you're removing your serve task, this line can be removed.

File: orko/package.json [hunk 3]
```diff
@@ -27,8 +21,6 @@
         "postcss-custom-selectors": "^4.0.1",
         "postcss-easy-import": "^3.0.0",
         "postcss-nested": "^3.0.0",
-        "rollup-plugin-babel": "^4.3.0",
-        "rollup-plugin-commonjs": "^9.1.8",
         "rollup-plugin-node-resolve": "^3.4.0",
```
Comment (aladage) [inline orko/package.json]: All of the rollup related lines can be removed.

**Comment by aladage** [comment]
I'm done reviewing what's been changed/removed, but also take a look at removing from what's still there, including:

- Remove /orko/_01-foot.mustache
- Remove /orko/main.js
- Remove patternlab reference in /orko/.gitignore
- Remove /skeletor/serve.config.js
- Remove /.babelrc
- Remove /.eslintrc
- Remove copy references to anything already removed from /orko.config.js

File: orko/package.json [hunk 4]
```diff
@@ -6,18 +6,12 @@
...
         "@deg-skeletor/plugin-copy": "^1.0.2",
         "@deg-skeletor/plugin-express": "^1.0.0",
-        "@deg-skeletor/plugin-patternlab": "^3.1.0",
         "@deg-skeletor/plugin-postcss": "^1.0.0",
         "@deg-skeletor/plugin-rollup": "^2.0.2",
         "@deg-skeletor/plugin-watch": "^1.0.0",
-        "@deg-skeletor/starterkit-mustache-default": "^1.0.1",
         "autoprefixer": "^8.6.5",
         "cssnano": "^3.10.0",
         "jest": "^23.6.0",
```
Comment (heaper) [inline orko/package.json]: Jest can be removed

File: skeletor.config.js [hunk 5]
```diff
@@ -4,8 +4,6 @@ module.exports = {
 			name: 'build',
```
Comment (heaper) [inline skeletor.config.js]: The entire build task can be removed, right?

**Comment by aladage** [comment]
Looks good to me

**Pull request merged**